Dockerfile

```diff
@@ -89,7 +89,7 @@ COPY . .
 
 # start building the final container
 FROM nemo-deps as nemo
-ARG NEMO_VERSION=1.0.0rc1
+ARG NEMO_VERSION=main
```

Comment:
This should revert to 1.0.0rc1

Reply:
why though? 1.0.0rc1 was never tested with 21.03 container?